docs/userguides/forking_networks.md

```diff
@@ -0,0 +1,81 @@
+# Forking Networks
+
+You can fork live networks in Ape.
+To do so, ensure you are using a provider plugin with forking features.
+Some options are:
+
+1. [ApeWorX/ape-foundry](https://github.com/ApeWorX/ape-foundry)
+2. [ApeWorX/ape-hardhat](https://github.com/ApeWorX/ape-hardhat)
+
+You can install one of these plugins by doing:
+
+```shell
+ape plugins install <foundry|hardhat>
+```
+
+Ensure you have also configured your upstream network (the network you are forking).
+For example, if forking `ethereum:mainnet` and using `alchemy`, set `alchemy` as the default mainnet provider:
+
+```yaml
+ethereum:
+  mainnet:
+    default_provider: alchemy
+```
+
+Now, you can start and connect to your forked-network:
+
+```yaml
+ape console --network ethereum:mainnet-fork:foundry
+```
+
+Learn more about setting up networks in the [the networks guide](./networks.html).
+
+## Forking Plugin Networks
```

Comment:
I am not sure it is possible fork custom networks right now in Ape, else I would also add a section for this.
That being said, it would be nice if we could because users are going to ask for this I am sure.

I need to come up with how to add that.